src/MusicPlaylistKernel.java

```diff
@@ -0,0 +1,86 @@
+
+import components.map.Map;
+import components.queue.Queue;
+import components.standard.Standard;
+
+/**
+ * Kernel Implementation for {@code MusicPlaylist} implemented on {@code Queue}
+ * with {@code Map}.
+ *
+ * @author Nijaa Nishanth
+ */
+public interface MusicPlaylistKernel extends Standard<MusicPlaylist> {
```

Comment:
Ah, wait I realize what's gone wrong. It's just this line. Ignore the comments below (or above depending on how they're shown).

Reply:
```suggestion
public class MusicPlaylist1 extends MusicPlayerSecondary {
```